src/Params/Controller.cpp

```diff
@@ -112,6 +114,7 @@ void Controller::defaults()
     NRPN.receive                 = 1;
 
     portamento.portamento        = 0;
+    portamento.automode          = 0;
```

Comment:
After trying out the code I think that setting automode to 1 by default makes the most sense. Any opposition to that change?

Reply:
Not much, it really is your call. My rationale was that automode modifies the basic portamento function and makes the behavior more intricate, and as such should be disabled by default as that reflects the simplest behavior. On the other hand, a rationale for the opposite would be that portamento is most likely used in the monophonic modes, where automode previously was the only behavior, and so for reasons of familiarity should be enabled by default.